maintenance/doMaintenance.php

```diff
@@ -124,14 +126,23 @@
 // Much much faster startup than creating a title object
 $wgTitle = null;
 
+\Wikia\Logger\WikiaLogger::instance()->info("Maintenance script $maintClass started.");
```

Comment:
I'd like all the messages texts to carry enough information to make some sens and in this case I think this is required to make any sense.